doc/website/blog/2022-03-14-announcing-preview-support-for-tidb.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Announcing preview support for TiDB
+author: Amit Shani
+authorURL: "https://github.com/hedwigz"
+authorImageURL: "https://avatars.githubusercontent.com/u/8277210?v=4"
+authorTwitter: itsamitush
+---
+
+Dear community,
+
+We [previously announced](https://entgo.io/blog/2022/01/20/announcing-new-migration-engine) the new migration engine - `Atlas`.
+With `Atlas`'s new design, it became easier than ever to add support for new databases for Ent.
+Today, I am happy to announce that a preview support for [TiDB](https://en.pingcap.com/tidb/) is now available, using the latest version of Ent with `Atlas` enabled.  
+For a quick `Hello World` with `Ent`+`TiDB`, follow the following steps:  
+1. Spin up a local TiDB server by using Docker:
+ ```shell
+ docker run -p 4000:4000 pingcap/tidb
+ ```
+ You should now have a running instance of TiDB listening on port 4000.
+
+2. Clone the example [`hello world` repository](https://github.com/hedwigz/tidb-hello-world):
+ ```shell
+ git clone https://github.com/hedwigz/tidb-hello-world.git
+ ```
+ 
+ Connecting Ent with TiDB is easy, just peek at `main.go`
+ ```go {1,7} title="main.go"
```

Comment:
Why do you highilght only lines 1-7? It looks weird :-D

Reply:
:rocket: